### PR TITLE
feat: add playwright integration

### PR DESCRIPTION
**Pull request by Zidious**
New java integration! 

[Rendered README](https://github.com/dequelabs/axe-core-maven-html/pull/206/files?short_path=c965fc0#diff-c965fc0a1aae58b7cdd9aa60904ee51ecc5732ff358db4174f41936378510826)

Playwright Java: https://playwright.dev/java/

☕

### CODE REVIEW

**Comment by Zidious** [comment]
Snapshot release: [nexus](https://oss.sonatype.org/#nexus-search;quick~deque)
image

CI pipeline: https://app.circleci.com/pipelines/github/dequelabs/axe-core-maven-html/501/workflows/0851c7fb-1cad-407e-93f0-2fe6c8cd55bb

File: playwright/src/main/java/com/deque/html/axecore/playwright/AxeBuilder.java [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+            AxeRuleOptions axeRuleOptions = new AxeRuleOptions();
+            axeRuleOptions.setEnabled(false);
+            disableRulesMap.put(rule, axeRuleOptions);
+          });
+      this.options.setRules(disableRulesMap);
+    } else {
+      throw new RuntimeException(
+          "disableRules list cannot be empty. "
+              + "Please see: https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md#rule-descriptions");
+    }
```
Comment (stephenmathieson) [inline playwright/src/main/java/com/deque/html/axecore/playwright/AxeBuilder.java:141]: Nit - flip the `if` so we can exit early. Doing so keeps the happy-path closer indented to the left, which tends to make for easier to read code.

```suggestion
     if (rules.isEmpty()) {
      throw new RuntimeException(
          "disableRules list cannot be empty. "
              + "Please see: https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md#rule-descriptions");
    }
    Map<String, AxeRuleOptions> disableRulesMap = new HashMap<>();
    rules.forEach(
        (rule) -> {
          AxeRuleOptions axeRuleOptions = new AxeRuleOptions();
          axeRuleOptions.setEnabled(false);
          disableRulesMap.put(rule, axeRuleOptions);
        });
    this.options.setRules(disableRulesMap);
```

Nice doc on this- https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88

File: playwright/src/main/java/com/deque/html/axecore/utilities/axeresults/AxeRuntimeException.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.deque.html.axecore.utilities.axeresults;
+
+public class AxeRuntimeException extends RuntimeException {
+  private static final long serialVersionUID = -123456789087654L;
```
Comment (stephenmathieson) [inline playwright/src/main/java/com/deque/html/axecore/utilities/axeresults/AxeRuntimeException.java:4]: Java is weird.

File: playwright/src/main/java/com/deque/html/axecore/playwright/AxeBuilder.java (hunk 1 above)
Comment (Zidious) [reply playwright/src/main/java/com/deque/html/axecore/playwright/AxeBuilder.java:141]: Oh, good article - I agree will update shortly

**Review by AdnoC — APPROVED** [review]

**Pull request merged**